otx/algorithms/classification/tasks/inference.py

```diff
@@ -347,22 +363,23 @@ def patch_color_conversion(pipeline):
                     patch_color_conversion(pipeline_step.transforms)
 
         assert "data" in config
-        for subset in ("train", "val", "test"):
+        for subset in ("train", "val", "test", "unlabeled"):
             cfg = config.data.get(subset, None)
             if not cfg:
                 continue
             if cfg.type == "RepeatDataset":
                 cfg = cfg.dataset
 
-            if self._multilabel:
-                cfg.type = "MPAMultilabelClsDataset"
-            elif self._hierarchical:
-                cfg.type = "MPAHierarchicalClsDataset"
-                cfg.hierarchical_info = self._hierarchical_info
-                if subset == "train":
-                    cfg.drop_last = True  # For stable hierarchical information indexing
             else:
-                cfg.type = "MPAClsDataset"
+                if self._multilabel:
```

Comment:
how about change if statement to elif statement and decreasing indent level of below elif and else statement?